ui/app/styles/utils/_bulma_variables.scss

```diff
@@ -17,6 +17,7 @@ $light-blue: #e2eafa;
 $dark-blue: #1563ff;
 $light-blue: rgb(218, 234, 247);
 $dark-green: #36ae40;
+$green-darker: darken($dark-green, 15%);
```

Comment:
Would `$darker-green` be better for consistency?

Little offshoot and just an observation, but this is where `$green-500`, `$green-600` so the numbering thing the design team are doing etc etc would help - means there is plenty of room between 500 and 600 to shim in different shades if we need them post original shades.`$vader-green`? 😄